pkg/utils/config/go.mod

```diff
@@ -0,0 +1,5 @@
+module nri/nri-plugins/pkg/utils/config
+
+go 1.19
```

Comment:
small nit: with go 1.19 already out of support, should we require minimum 1.20?

Reply:
I used 1.19 because the entire project is on 1.19. If you don't mind, I will switch to 1.20, which is my own preference too.

Reply:
Let's do that in a separate PR.